nemo/collections/asr/parts/submodules/rnnt_beam_decoding.py

```diff
@@ -39,6 +39,7 @@
 from nemo.core.classes import Typing, typecheck
 from nemo.core.neural_types import AcousticEncodedRepresentation, HypothesisType, LengthsType, NeuralType
 from nemo.utils import logging
+from nemo.collections.asr.modules.rnnt import RNNTDecoder, StatelessTransducerDecoder
```

Comment:
Wait, this should not be imported at top level, it can lead to circular import. Import this only at the location that checks are needed so that's dependency resolution can act in a delayed manner

Reply:
Sure.